server/planning/common.py

```diff
@@ -135,6 +135,27 @@ class PostStates(NamedTuple):
 TO_BE_CONFIRMED_FIELD = "_time_to_be_confirmed"
 TO_BE_CONFIRMED_FIELD_SCHEMA = {"type": "boolean"}
 
+EVENT_IGNORED_FIELDS = {
```

Comment:
The name `EVENT_IGNORED_FIELDS` does not provide context as to when these fields are ignored.
It would provide more meaning if the name indicates that these are the fields that are ignored when duplicating an Event